### ISSUE TITLE
web: non-afk time is not being properly tallyed up

### ISSUE THREAD

**Issue by kewde** [ActivityWatch/activitywatch#687]
- [x] I am on the [latest](https://github.com/ActivityWatch/activitywatch/releases/latest) ActivityWatch version.
 - [x] I have searched the issues of this repo and believe that this is not a duplicate.

- **OS name and version**: Windows 10
- **ActivityWatch version**: v0.11.0

## Describe the bug
The timeline barchart + time active of the activity view is under-reporting the actual hours.
![activitywatchbug](https://user-images.githubusercontent.com/13770485/141154255-25dfcc5f-0731-4908-bbb3-01ad78cb7314.png)
![activitywatchbughalffix](https://user-images.githubusercontent.com/13770485/141154564-4737054b-8ff2-4fea-af11-ef57b5adccae.png)

## To Reproduce
Count up non-afk time for a day, compare it with what's shown in the "Activity view" under "time active".

## Expected behavior
All Non-AFK time being properly tallyed

**Comment by kewde** [comment]
I noticed that the returned output of the HTTP request for `/query` doesn't include any events for the hour 4 PM.
Either the query is broken or the query engine behind it is malfunctioning.

The `13`th element (index 12)  represents the 4PM column because Activity Watch starts counting from 4 AM. As you can see in the screenshot, it's empty.
![activitywatchbugquery](https://user-images.githubusercontent.com/13770485/141157960-f7285647-24f7-459c-8516-b07b9704a972.png)

**Comment by ErikBjare** [comment]
Looks like a duplicate of https://github.com/ActivityWatch/activitywatch/issues/609, which is fixed in master (and available in nightly releases, see #507).

Not sure about the non-afk time not adding up though (might be related to the above issue, but not sure). If you could try a nightly build and report back that would be helpful!

**Comment by kewde** [comment]
@ErikBjare 
The barchart has been fixed by switching over to Rust.

When I tallyed up the times again and my initial calculation was wrong. I'm still getting a few minutes of difference between what's show in 'Time Active' and manually adding up the non-afk events in the timeline but it's in the order of minutes, which is acceptable to me.

I'll give the nightlies a go on my personal machine, thanks for the quick response!
